sass/directives/02_base_components/_modals.scss

```diff
@@ -31,3 +31,16 @@
     text-decoration: none;
   }
 }
+
+@mixin close-x-overflow {
+  width: 25px;
```

Comment:
I'm going to create another variable for this. The two are unrelated however despite being the same size.